quickstarts/microsoft.compute/vm-simple-windows/createUiDefinition.json

```diff
@@ -63,150 +63,42 @@
                         "toolTip": "The OS Version for the VM Image",
                         "constraints": {
                             "allowedValues": [
-                                {
-                                    "value": "2008-R2-SP1",
-                                    "label": "2008-R2-SP1"
-                                },
-                                {
-                                    "value": "2008-R2-SP1-smalldisk",
-                                    "label": "2008-R2-SP1-smalldisk"
-                                },
-                                {
-                                    "value": "2012-Datacenter",
-                                    "label": "2012-Datacenter"
-                                },
-                                {
-                                    "value": "2012-datacenter-gensecond",
-                                    "label": "2012-datacenter-gensecond"
-                                },
-                                {
-                                    "value": "2012-Datacenter-smalldisk",
-                                    "label": "2012-Datacenter-smalldisk"
-                                },
-                                {
-                                    "value": "2012-datacenter-smalldisk-g2",
-                                    "label": "2012-datacenter-smalldisk-g2"
-                                },
```

Comment:
Just curious, why are these removed? Comment applies everywhere a valid sku was removed

Reply:
Trusted Launch is not supported for Windows Server 2012/2012 R2.
It is only supported for Gen2 OS images. Hence all Gen1 OS images were removed.

Reply:
I was referring to the valid G2 skus. I see that there are G2 (gen 2) skus removed.